builtin/providers/google/resource_compute_vpn_tunnel.go

```diff
@@ -72,6 +72,14 @@ func resourceComputeVpnTunnel() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"remote_traffic_selector": &schema.Schema{
+				Type:     schema.TypeSet,
```

Comment:
Three reasons:

-  local_traffic_selector also uses a set (this by itself is not an excuse).
- Cloud VPN doesn't care about the order of the traffic selectors, so even if a remote side has the local traffic selector configured in the specific order as "A, B", and on Cloud VPN side the remote traffic selector has been configured as "B, A", Cloud VPN will still successfully negotiate the child SA.
- In general, the main reason to use a set instead of a list in terraform is that if the user changes the definition from ["A", "B"] to ["B", "A"], terraform does not consider this as a delta and will not recreate the resource (or update, for resources that support that). 

I'll update the PR with some tests.